presto-orc/src/main/java/com/facebook/presto/orc/stream/OrcInputStream.java

```diff
@@ -83,6 +83,7 @@ public OrcInputStream(OrcDataSourceId orcDataSourceId, FixedLengthSliceInput sli
     public void close()
     {
         current = null;
+        // todo what should we do here?
```

Comment:
As discussed offline, this `close` method is not called at all, and `OrcRecordReader::advanceToNextStripe` already closes the`currentStripeSystemMemoryContext`. Therefore, the memory accounting calls in this method are not needed. We can add a comment here and remove these calls.